website/src/content/docs/pt-br/reference/vscode.mdx

```diff
@@ -1,76 +1,78 @@
 ---
-title: VSCode extension
-description: Notes about the Biome's VSCode extension
+title: Extensão VSCode
+description: Notas sobre a extensão do Biome para o VSCode
 ---
-# Biome VS Code Extension
+# Extensão do Biome para o VS Code
 
-[Biome](https://biomejs.dev/) unifies your development stack by combining the functionality of separate tools. It uses a single configuration file, has fantastic performance, and works with any stack. This extension brings Biome to your editor so that you can:
+[O Biome](https://biomejs.dev/) unifica sua pilha de desenvolvimento combinando a funcionalidade de ferramentas separadas. Ele usa um único arquivo de configuração, possui desempenho fantástico e funciona com qualquer pilha. Esta extensão traz o Biome para o seu editor para que você possa:
 
-- Format files *on save* or when issuing the *Format Document* command
-- See lints while you type and apply code fixes
-- Perform refactors
+- Formatar arquivos *ao salvar* ou ao emitir o comando *Formatar Documento*
+- Ver lints enquanto digita e aplicar correções de código
+- Realizar refatorações
 
-## Installation
+## Instalação
 
-You can install the code extension by heading to the extension's [Visual Studio Code Market Place page](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) or from within VS Code by either:
+Você pode instalar a extensão do código acessando a [página do Visual Studio Code Market Place da extensão](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) ou a partir do VS Code:
 
-- Open the *extensions* tab (_View_ → _Extensions)_ and search for Biome.
-- Open the _Quick Open Overlay_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> or _Go -> Go to File_), enter `ext install biomejs.biome`, and hit enter.
+- Abra a aba *extensões* (_Exibir_ → _Extensões)_ e procure por Biome.
+- Abra a _Sobreposição de Abertura Rápida_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> ou _Ir -> Ir para Arquivo_), digite `ext install biomejs.biome` e pressione enter.
 
-## Getting Started
+## Primeiros Passos
 
-### Default Formatter
+### Formatador Padrão
 
-Configure Biome as the default formatter for supported files to ensure that VS Code uses Biome over other formatters that you may have installed. You can do so by opening a JavaScript or TypeScript and then:
+Configure o Biome como o formatador padrão para arquivos suportados para garantir que o VS Code use o Biome em vez de outros formatadores que você possa ter instalado. Você pode fazer isso abrindo um JavaScript ou TypeScript e depois:
 
-- Open the Command Palette (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> or View → Command Palette)
-- Select _Format Document With…_
-- Select _Configure Default Formatter…_
-- Select Biome
+- Abra a Paleta de Comandos (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> ou Exibir → Paleta de Comandos)
```

Comment:
```suggestion
- Abra a Paleta de Comandos (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> ou Ver → Paleta de Comandos)
```
 I noticed that in the Portuguese version of vs code, the menu is called "Ver" and not "Exibir"

Reply:
nice catch, I just translated according to my experience.